### PR TITLE
[FLOC-1622] Manual cli test floc 1622

### PR DESCRIPTION
**Pull request by adamtheturtle**
Fixes https://clusterhq.atlassian.net/browse/FLOC-1622

### CODE REVIEW

**Comment by exarkun** [comment]
Has a merge conflict with master, can you resolve?

File: docs/gettinginvolved/client-testing.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+.. prompt:: bash $
+
+   docker run -i -t ubuntu /bin/bash
+
+or:
+
+.. prompt:: bash $
+
+   docker run -i -t fedora:20 /bin/bash
```
Comment (exarkun) [inline docs/gettinginvolved/client-testing.rst]: Probably `fedora:22` :/

Comment (adamtheturtle) [inline docs/gettinginvolved/client-testing.rst]: The docs say Fedora 20 at http://doc-dev.clusterhq.com/using/installing/index.html#other-linux-distributions and http://doc-dev.clusterhq.com/gettingstarted/index.html#installation so I chose Fedora 20.

**Comment by adamtheturtle** [comment]
> Has a merge conflict with master, can you resolve?

Done

File: docs/gettinginvolved/client-testing.rst (hunk 1 above)
Comment (exarkun) [inline docs/gettinginvolved/client-testing.rst]: So sad.  Okay.

Can you file an issue for factoring the `20` out of all these different places so we can indicate our supported Fedora version in just one place?

**Comment by exarkun** [comment]
Thanks.  Looks good.  Please merge.

File: docs/gettinginvolved/client-testing.rst (hunk 1 above)
Comment (adamtheturtle) [inline docs/gettinginvolved/client-testing.rst]: I added a comment for https://clusterhq.atlassian.net/browse/FLOC-483 as I think that this suggestion comes under that title.

**Pull request merged**